src/main/java/org/graphqlcrudjava/DatabaseAnalyzer.java

```diff
@@ -0,0 +1,158 @@
+package org.graphqlcrudjava;
+
+import org.graphqlcrudjava.model.Attribute;
```

Comment:
I think we can drop java. Unless we do some kotlin stuff. Actually do you like kotlin?? :)

Reply:
> I think we can drop java. Unless we do some kotlin stuff. Actually do you like kotlin?? :)

Not sure about it because I have never worked in kotlin. 😅

Reply:
Yep.. Semi joke. Java will be prefered for this anyway.